xbmc/cores/RetroPlayer/messages/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+set(MESSAGES savestate.fbs
+)
+
+core_add_messages(retroplayer_messages)
+set(EXTRA_INCLUDES ${EXTRA_INCLUDES} PARENT_SCOPE)
```

Comment:
@Rechi I need to pass the generated header's directory to the root cmake file. Is this an OK way to do that?

Reply:
No, but haven't checked how to do it.